### ISSUE TITLE
How to build and contribute a new destination

### ISSUE THREAD

**Issue by johnlafleur** [airbytehq/community-content#4]
Some potential contributors prefer to see video tutorials instead of documentation articles, and see people actually show them exactly how to do things. We could also use the video to improve the documentation, and an article on external publications to show how building your own integration is going to be trivial using Airbyte, which is great for awareness towards our mission.

**Comment by DominusKelvin** [comment]
@avaidyanatha is this still blocked?

**Comment by avaidyanatha** [comment]
@DominusKelvin 

Nope! We have the two guides here that can be followed in the video:

https://docs.airbyte.io/connector-development/tutorials/building-a-python-destination
https://docs.airbyte.io/connector-development/tutorials/building-a-java-destination

**Comment by DominusKelvin** [comment]
> @DominusKelvin 
> 
> 
> 
> Nope! We have the two guides here that can be followed in the video:
> 
> 
> 
> https://docs.airbyte.io/connector-development/tutorials/building-a-python-destination
> 
> https://docs.airbyte.io/connector-development/tutorials/building-a-java-destination

Got you
